icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2462,14 +2464,18 @@ class U_I18N_API FormattedNumber : public UMemory {
 #endif  /* U_HIDE_DEPRECATED_API */
 
     /**
-     * Returns a UnicodeString representation of the formatted number.
+     * @copybrief FormattedValue::toString()
      *
-     * @param status
-     *            Set if an error occurs while formatting the number to the UnicodeString.
-     * @return a UnicodeString containing the localized number.
      * @draft ICU 62
      */
-    UnicodeString toString(UErrorCode& status) const;
+    UnicodeString toString(UErrorCode& status) const U_OVERRIDE;
+
+    /**
+     * @copybrief FormattedValue::toTempString()
```

Comment:
I'm not sure what the best answer is here.

It's more common in C++ than in Java to look at the header itself rather than the generated docs, and seeing @copydoc as a description is not helpful. But copying the content around, and having some copies grow stale over time, is not good either.

Reply:
I put it on the agenda.

Reply:
Changed as discussed.